notifications/01-27-22-1300.md

```diff
@@ -0,0 +1,42 @@
+**What is happening?**
+
+Migrate NEW SIS (The Secure Internet Service) Firewalls from OLD WAN R01 to New Arista Switches in Calgary and Kamloops
+
+Other tickets references:
+- TIB [KB0031869](https://ociomysc.service-now.com/nav_to.do?uri=u_kb_template_technical_information_bulletin.do?sys_id=34d5585bdb480550fa86193813961968)
+- MySC CHG0047385
+- theHUB CHG0032439
+
+One firewall will be migrated at a time, Calgary FW first, Interior FW second. This has the potential to cause two disruptions to traffic of up to 15 minutes each while the new firewall is configured.
+
+This will cause two brief disruptions to Internet traffic for all servers and applications when traffic is failed over from one datacentre to the other.
```

Comment:
```suggestion
This will cause two brief disruptions to Internet traffic for all servers and applications hosted in the data centers when traffic is failed over from one datacentre to the other.
```